app/assets/stylesheets/helpers/_core.scss

```diff
@@ -298,38 +298,6 @@ h4 {
   }
 }
 
-#wrapper.businesslink #legacy-sources,
-#wrapper.directgov #legacy-sources {
-  display: block;
-}
-
-#wrapper.businesslink #legacy-sources li.businesslink {
-	background-image: image-url("businesslink-logo.png");
```

Comment:
And..now removed!  https://github.com/alphagov/performanceplatform-selfserve have their own local copy in the bower-components folder :), thanks again @alextea for pointing this out!